### PR TITLE
Add get followers test

### PR DESCRIPTION
**Pull request by eucyt**
`client.getFollowers(null, 99)` must issue a request url like `/v2/bot/followers/ids?limit=99`. There must not be start as request parameter. This change adds a test to verify this.

### CODE REVIEW

**Comment by eucyt** [comment]
Now I confirmed new test added!
image

File: .github/workflows/php-checks.yml [hunk 1]
```diff
@@ -46,11 +46,6 @@ jobs:
           export PATH=$PATH:~/bin/openapitools/
           OPENAPI_GENERATOR_VERSION=${{ env.OPENAPI_GENERATOR_VERSION }} openapi-generator-cli version
 
-      - name: Generate code
-        run: |
-          export PATH=$PATH:~/bin/openapitools/
-          bash tools/gen-oas-client.sh
-
```
Comment (eucyt) [inline .github/workflows/php-checks.yml:53]: I think that this code is unnecessary. This is because it causes a discrepancy with the actual commit content, leading to tests being conducted in that state, and the code generated by this script is not reflected in the PR. Since the differences generated by oas are submitted in a separate PR by diff-check.yaml, it seems unnecessary to do it here.

Comment (Yang-33) [reply .github/workflows/php-checks.yml:55]: Since there is a possibility of deletion here(https://github.com/line/line-bot-sdk-php/blob/e1609278c0cfc5c3bb8216736548292d806995ea/.github/workflows/diff-check.yml#L49) as well, I cannot accept deleting this.

Isn't the deletion of existing code during code generation itself problematic?

There are two options for a solution:

1. Remove only the script that deletes existing code, but keep the call to `gen-oas-client.sh`.
2. Move the location where test files are placed to somewhere other than `./src/clients/<project>.`

**Review by Yang-33 — COMMENTED** [review]
commented

File: .github/workflows/php-checks.yml (hunk 1 above)
Comment (eucyt) [reply .github/workflows/php-checks.yml:55]: > Isn't the deletion of existing code during code generation itself problematic?

I agree. Furthermore, I think the problem is that it is **difficult to notice when existing code is deleted or also edited**. While code that is deleted or edited by diff-check.yml is displayed as a difference in the PR, in php-checks.yml (CI) that is not immediately apparent. I am concerned about whether it is okay to conduct tests in a state where the code has been altered from the intended state.

Out of the options mentioned above, I think option 1 is preferable. 
However, this would mean that tests are conducted in a state where there are newly added or edited files that are not reflected in the differences shown in the PR. Is this acceptable?

Comment (eucyt) [reply .github/workflows/php-checks.yml:55]: The reason I allowed the execution of gen-oas-client.sh in diff-check.yml is that the deleted or edited code is displayed as a difference in the PR, ensuring that tests are not conducted in an unintended state. However, I agree, it is not ideal for the code to be deleted or edited in the first place :sweat_smile:

Comment (Yang-33) [reply .github/workflows/php-checks.yml:55]: > However, this would mean that tests are conducted in a state where there are newly added or edited files that are not reflected in the differences shown in the PR. Is this acceptable?

No it's not acceptable.

In this case, it would be appropriate to check for any differences using a separate job.
When there are differences other than those in the PR, marking the CI results as failed will alert us that there is invisible extra diff.

Comment (eucyt) [reply .github/workflows/php-checks.yml]: I've implemented it with option 2 after all.

- If I eliminate `rm -rf $REPO_ROOT_DIR/src/clients/$schema`, we won't be able to update the deleted items within the auto-generated code. In other words, no code other than the auto-generated one should be placed under `clients/$schema`.
- Generally, in PHP, tests are often placed in tests/unit at the same level as the src directory. [FYI](https://docs.phpunit.de/en/10.5/organizing-tests.html)

Comment (Yang-33) [reply .github/workflows/php-checks.yml]: thanks!

File: .github/workflows/php-checks.yml [hunk 2]
```diff
@@ -6,6 +6,58 @@ on:
   merge_group:
 
 jobs:
+  diff-check:
```
Comment (Yang-33) [inline .github/workflows/php-checks.yml:9]: did you check if it works as expected?

**Review by Yang-33 — APPROVED** [review]
LGTM though I left 1 comment(https://github.com/line/line-bot-sdk-php/pull/604#discussion_r1710637748)

File: .github/workflows/php-checks.yml (hunk 2 above)
Comment (eucyt) [reply .github/workflows/php-checks.yml:9]: It works!
https://github.com/line/line-bot-sdk-php/actions/runs/10313168121/job/28549497048?pr=604
